client/analytics/settings/index.js

```diff
@@ -204,12 +197,23 @@ class Settings extends Component {
 }
 
 export default compose(
-	withSettings( 'wc_admin' ),
+	withSelect( select => {
+		const { isResolving, getSetting, getLastSettingsErrorForGroup } = select( SETTINGS_STORE_NAME );
+		return {
+			isRequesting: isResolving( 'setSetting', [ 'wc_admin' ] ),
+			settings: getSetting( 'wc_admin', 'wcAdminSettings' ),
```

Comment:
>Where is `wcAdminSettings` coming from?

Its an object on the `wcSettings` object. You can see it in the console:

```js
wcSettings.wcAdminSettings.woocommerce_actionable_order_statuses
// ["processing", "on-hold"]
```
`wcAdminSettings` is in fact the settings object that powers the settings page. 

I agree with you that it should be more like this:

```js
const actionableOrderStatuses = getSetting( 'wc_admin', 'woocommerce_actionable_order_statuses' );
```
but that doesn't work as its currently structured. Despite the redundant naming, I'd like to avoid manipulating how this object is stored because its used by onboarding. I could refactor the store to look for keys multiple levels deep, but I'd rather stick as closely to the current usage and revisit how this object is stored when I address how OBW interacts with the result of this work.